front/tests/005-operational-studies.spec.ts

```diff
@@ -93,15 +93,6 @@ test.describe('Testing if all mandatory elements simulation configuration are lo
 
     await scenarioPage.checkPathfindingDistance('34.000 km');
 
-    // TODO: move this test in his own file
```

Comment:
@Maymanaf like discussed can you check the test part :)

Reply:
This part of the tests covers the old component and is no longer needed, aas we will have two tests that address the new behavior:

- A new test: https://github.com/OpenRailAssociation/osrd/issues/8647
- A refactor of test 005: https://github.com/OpenRailAssociation/osrd/issues/8648